AndroidManifest.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Let's remove this (see comment)

Reply:
I'm having a hard time understanding what you  want me to remove?

Reply:
see comment on MainActivity.java